enterprise/backend/test/metabase_enterprise/data_editing/api_test.clj

```diff
@@ -1201,29 +1201,62 @@
                           [2 "seeya, world!" nil nil                   nil]]
                          (->> (table-rows table-id)
                               (sort-by first)))))
-                ;; This is failing because id is auto-incremented and does not accept a value
-                ;; we need to change the tests so that the upsert is conditioned on columns other than PK
-                ;; This should match with how it's intended to use too. but we haven't made the mapping for row-key
-                ;; so this is commented out for now
```

Comment:
Pretty sure this will still fail on some of the drivers

Reply:
we're targeting postgres and h2 right?

Reply:
I managed to reproduce this failure with either h2 or postgres, but it might depend on your configuration (so maybe just my machine and not CI)